README.md

```diff
@@ -37,7 +39,8 @@ You can also refer to the project's own [Dockerfile.dapper](Dockerfile.dapper) a
 
 ### Makefile
 
-The Makefile should include targets to run everything in Dapper. They're defined in [Makefile.dapper](Makefile.dapper) and can be copied as-is and included. To use Shipyard's target, simply include the [Makefile.inc](Makefile.inc) file in your own Makefile.
+The Makefile should include targets to run everything in Dapper. They're defined in [Makefile.dapper](Makefile.dapper) and can be copied
+as-is and included. To use Shipyard's target, simply include the [Makefile.inc](Makefile.inc) file in your own Makefile.
```

Comment:
```suggestion
The Makefile should include targets to run everything in Dapper.
They're defined in [Makefile.dapper](Makefile.dapper) and can be copied as-is and included.
To use Shipyard's target, simply include the [Makefile.inc](Makefile.inc) file in your own Makefile.
```

It’s better to break long lines following sentence structure; this reduces the impact of future changes (which tend to be sentence-bound) in diffs.

Reply:
Interesting idea. I agree it might make diffs better, but I for one plan on using automation to auto-wrap lines, and I don't see a good option for automatically following this pattern. For this tiny README it's not a problem, but I also have in mind the massive LL PR still pending for the website.